src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -225,34 +225,56 @@ def gen_image_map(config) {
         config.runs_on_dockers.each { item ->
 
             def dfile = item.clone()
-            dfile.arch = dfile.arch ?: arch
+            config.logger.debug("run on dockers item: " + dfile)
+            config.logger.debug("run on dockers item.name: " + dfile.name)
+            config.logger.debug("run on dockers item.enable: " + dfile.enable)
```

Comment:
AFAIK this is old logger API. You need to use logging with integer level.

Reply:
why line 228 was deleted?

Reply:
228 was not deleted. just moved to 245